Makefile

```diff
@@ -172,3 +180,55 @@ gcb-signer-image: out/gcb-signer-image
 gcb-signer-push-image: gcb-signer-image
 	docker push $(REGISTRY)/kritis-gcb-signer:$(IMAGE_TAG)
 
+# Steps to create a new test cluster on an GCP project that has already had a Kritis setup.
+.PHONY: create-integration-local
+create-integration-local:
+	gcloud --project=$(GCP_PROJECT) container clusters describe $(TEST_CLUSTER) >/dev/null \
+		|| gcloud --project=$(GCP_PROJECT) container clusters create $(TEST_CLUSTER) \
+		--num-nodes=2 --zone=$(GCP_ZONE)
+	gcloud --project=$(GCP_PROJECT) container clusters get-credentials $(TEST_CLUSTER)
+	mkdir -p $(dir $(GAC_CREDENTIALS_PATH))
+	test -s $(GAC_CREDENTIALS_PATH) \
+		|| gcloud --project=$(GCP_PROJECT) iam service-accounts keys \
+		create $(GAC_CREDENTIALS_PATH) --iam-account kritis-ca-admin@${GCP_PROJECT}.iam.gserviceaccount.com
+	kubectl create serviceaccount --namespace kube-system tiller
+	kubectl create clusterrolebinding tiller-cluster-rule \
+		  --clusterrole=cluster-admin \
+		    --serviceaccount=kube-system:tiller
+	helm init --wait --service-account tiller
+	gcloud -q container images add-tag \
+		gcr.io/kritis-tutorial/acceptable-vulnz@sha256:2a81797428f5cab4592ac423dc3049050b28ffbaa3dd11000da942320f9979b6 \
+		gcr.io/$(GCP_PROJECT)/acceptable-vulnz:latest
+	gcloud -q container images add-tag \
+		gcr.io/kritis-tutorial/java-with-vulnz@sha256:358687cfd3ec8e1dfeb2bf51b5110e4e16f6df71f64fba01986f720b2fcba68a \
+		gcr.io/$(GCP_PROJECT)/java-with-vulnz:latest
+	gcloud -q container images add-tag \
+		gcr.io/kritis-tutorial/nginx-digest-whitelist:latest \
+		gcr.io/$(GCP_PROJECT)/nginx-digest-whitelist:latest
+	gcloud -q container images add-tag \
+		gcr.io/kritis-tutorial/nginx-no-digest-breakglass:latest \
+		gcr.io/$(GCP_PROJECT)/nginx-no-digest-breakglass:latest
+	gcloud -q container images add-tag \
+		gcr.io/kritis-tutorial/nginx-no-digest:latest \
+		gcr.io/$(GCP_PROJECT)/nginx-no-digest:latest
+
+# Fully setup local integration testing.
+.PHONY: setup-integration-local
+setup-integration-local: build-push-test-image create-integration-local
```

Comment:
I think build-push-test-image should be moved over to integration local.
I want to iterate over kritis code and run int test,  running setup-integration-local multiple times will create an error for  already set up cluster.
Ideally, i would want to my dev flow to be: 
```
make setup-integration-local 

make integration-local
make integration-local 
make integration-local
...
```
Right now,  i will have to 
```
make setup-integration-local
make build-push-test-image integration-local

```

Reply:
Good idea. Done!